src/views/datasources/dataimportcron/details/DataImportCronManageModal/DataImportCronManageModal.tsx

```diff
@@ -94,18 +93,13 @@ export const DataImportCronManageModal: React.FC<DataImportCronManageModalProps>
         </StackItem>
         <StackItem>
           <Form>
-            <FormGroup fieldId="dataimportcron-manage-source-type" label={t('Source type')}>
-              <TextInput
-                id="dataimportcron-manage-source-type-input"
-                isDisabled
-                isReadOnly
-                value={t('Registry')}
-              />
-            </FormGroup>
             <FormGroup
               fieldId="dataimportcron-manage-source-url"
               label={t('URL')}
```

Comment:
We could use the label 'Registry URL' here like in [CreateDataSourceForm.tsx](https://github.com/kubevirt-ui/kubevirt-plugin/pull/775/files#diff-1619ba701215e1e29e53db9b6a59d1181c3459b28b62cab3d990c598ec5c7517)

Reply:
Thanks!
fixed @upalatucci